src/RNGestureHandlerModule.web.ts

```diff
@@ -88,13 +90,22 @@ export default {
   },
   attachGestureHandler(
     handlerTag: number,
-    newView: number,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    newView: any,
     _actionType: ActionType,
     propsRef: React.RefObject<unknown>
   ) {
+    if (
+      !(newView instanceof HTMLElement || newView instanceof React.Component)
```

Comment:
Now that I think about it, I'm not sure whether `svg` inherits after `HTMLElement`. Could you check that?